libraries/AP_HAL_Linux/Perf.cpp

```diff
@@ -169,6 +169,12 @@ void Perf::end(Util::perf_counter_t pc)
         perf.max = elapsed;
     }
 
+    if (!perf.count) {
```

Comment:
Well it is a uint64_t I think this will never wrap and if it happen the perf.total will happen first and on this case the others values will be invalid.